### ISSUE TITLE
AttributeError: 'lxml.etree._ElementTree' object has no attribute '_root'

### ISSUE THREAD

**Issue by agitator** [collective/collective.vdexvocabulary#5]
plone 4.3.4 with imsvdex 1.1

```

    directory=directory
  File "/Users/dev/workspace/project/eggs/collective.vdexvocabulary-0.2.1-py2.7.egg/collective/vdexvocabulary/meta.py", line 63, in base_directive
    registerfile(_context, os.path.join(directory, filename), cls)
  File "/Users/dev/workspace/project/eggs/collective.vdexvocabulary-0.2.1-py2.7.egg/collective/vdexvocabulary/meta.py", line 39, in registerfile
    vocabulary_factory = cls(filename)
  File "/Users/dev/workspace/project/eggs/collective.vdexvocabulary-0.2.1-py2.7.egg/collective/vdexvocabulary/treevocabulary.py", line 127, in __init__
    self.vocabulary = VdexTreeVocabulary(filename, default_lang)
  File "/Users/dev/workspace/project/eggs/collective.vdexvocabulary-0.2.1-py2.7.egg/collective/vdexvocabulary/treevocabulary.py", line 88, in __init__
    rootelement = self.vdex.tree._root
zope.configuration.xmlconfig.ZopeXMLConfigurationError: File "/Users/dev/workspace/project/parts/instance/etc/site.zcml", line 15.2-15.55
    ZopeXMLConfigurationError: File "/Users/dev/workspace/project/parts/instance/etc/package-includes/002-client.site-configure.zcml", line 1.0-1.58
    ZopeXMLConfigurationError: File "/Users/dev/workspace/project/src/client.site/konfmusic/site/configure.zcml", line 42.2-42.50
    AttributeError: 'lxml.etree._ElementTree' object has no attribute '_root'
```

looks like some update for lxml (introduced by imsvdex) is necessary
workaround for now: pin imsvdex to 1.0

**Comment by jensens** [comment]
looks like #4 fixes this

**Comment by saily** [comment]
Closed by merging #4 into master. @agitator can you please make a new release or add me to the maintainers on pypi. thanks.
